ush/machine/hera.yaml

```diff
@@ -32,3 +32,40 @@ platform:
   FIXsfc: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_sfc_climo
   FIXshp: /scratch2/BMC/det/UFS_SRW_App/develop/NaturalEarth
   EXTRN_MDL_DATA_STORES: hpss aws nomads
+
+# RRFS input obs./data
+RAP:
+  local:
+    file_path: /scratch2/BMC/public/data/grids/rap/obs
+    file_names:
+      obs:
+        - "{yyyymmddhh}.rap.t{hh}z.prepbufr.tm00"
```

Comment:
These default to the filenames under hpss when not provided to retreive_data.py, so they should be left out to avoid duplicate information.

Reply:
this has been corrected. my remote and local branches were out of sync and not all of my updates were pushed to this remote. i apologize for that lapse.